### DISCUSSION TITLE
Potentially historically accurate bigger monument (Thermae)

### DISCUSSION THREAD

**Discussion by q6276270** [Keriew/augustus#607] [Ideas]
In GamerZakh's [latest](https://www.youtube.com/watch?v=gwMwRgaTHVk) Thera Playthrough there was a suggestion of possibly bigger monuments. I recently read an architecture book which also covered a bit of roman architecture and they mentioned Thermae (or Baths) as sometimes as big as smaller cities inside bigger cities. An advantage is they could be made to be pretty modular.

Unfortunately, I don't have any idea for an interesting and balanced bonus they could provide. Also I do not yet have an idea how they would get along well with the existing bathhouses.

**Comment by MHCreations** [comment]
Like temples perhaps they could be a "bigger bath" if someone can provide art work / graphics for the building it can be 3 x 3 or perhaps larger and limited to just 1 per city (?) ; rather like a Pantheon in-game or similar; its walkers would walk further and give a wider health benefit?    

There is a graphic of a large Thermae (Baths of Caracalla) here: https://www.globalsecurity.org/military/world/spqr/rome-baths-caracalla.htm

**Reply by PaxBotanica** [reply]
I really want to see this. Many baths in Rome were massive land area structures that rivalled or even the dwarfed the Colosseums footprint. I think it should be a necessary requirement for large villa and above. They could also offer building range or city wide health benefits to reduce plague.
